Competition2017/src/competition/subsystems/shooter_belt/commands/FireTracerMode.java

```diff
@@ -0,0 +1,27 @@
+package competition.subsystems.shooter_belt.commands;
+
+import competition.subsystems.shooter_belt.ShooterBeltSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class FireTracerMode extends BaseCommand {
```

Comment:
I would rename this to `RunBeltSlowlyCommand`, as there's nothing really "tracer-like" about this, since it just operates belt